.travis.yml

```diff
@@ -1,15 +1,16 @@
 # Configuration script for Travis-CI
 language: python
+dist: xenial
 python:
   - 2.7
+  - 3.6
+  - 3.7
 matrix:
   include:
-    - python: 2.7
-      name: "2.7"
     - python: 2.7
```

Comment:
I think it would be good to do the flake8 also with Python 3.7 if possible.

Reply:
> I think it would be good to do the flake8 also with Python 3.7 if possible.

Why?

Does flake8 produce different warnings on different Python versions?  (Other than syntax errors resulting from the `ast` module, but there should be none of those if the codebase is 2 & 3 compatible.)

Reply:
Flake8 on Python 3 typically raises NameErrors as `basestring`, `unicode` or SyntaxErrors as you mentioned. This is especially true for untested paths, which will not show up unless actually used. 

Apart from that, this would make it more future compatible as we aim for Python 3, but that is more a personal preference.

Reply:
Okay, I'll switch to Python 3.7 for the Flake8 tests.